src/views/ReviewConfirm.vue

```diff
@@ -153,6 +149,7 @@ export default class ReviewConfirm extends Mixins() {
   }
 
   /** Handler for setting DateTime. */
+  // TODO: fix type (should be Date)
```

Comment:
This should be fixed before we deploy to Prod.

If you're in this area (and have a bit of time), please fix this.